torch/_export/passes/replace_quantized_ops_with_standard_ops_pass.py

```diff
@@ -76,20 +76,20 @@ def get_dequantized(
     if qscheme is torch.per_tensor_affine:
         return dequantize_per_tensor(
             val,
-            scale,
-            zero_point,
-            qmin,
-            qmax,
+            scale,  # type: ignore[arg-type]
```

Comment:
FYI, seems like quantize_per_tensor and dequantize_per_tensor has incorrect typing.

Reply:
Agreed.  We should fix that eventually - but this PR is about annotating library.impl which is a bigger issue since that will strip the types off anything that is annotated with it (like dequantize_per_tensor).